detox/android/detox/src/main/java/com/wix/detox/reactnative/idlingresources/timers/DefaultIdleInterrogationStrategy.kt

```diff
@@ -70,25 +70,15 @@ class DefaultIdleInterrogationStrategy
             // Should have been handled by DelegatedIdleInterrogationStrategy.createIfSupported() but seems the new TimingModule class
             // was released without the awaited-for "hasActiveTimersInRange()" method.
             try {
-                val timingClass: Class<NativeModule> = Class.forName("com.facebook.react.modules.core.TimingModule") as Class<NativeModule>
-                if (!reactContext.hasNativeModule(timingClass)) {
-                    return null
-                }
-
-                val timingModule = reactContext.getNativeModule(timingClass)
```

Comment:
red is good

Reply:
Do you mean, this change will break RN < 0.62?

Reply:
no - just moved this repeating cumbersome reflection logic to a helper class, for reuse.